### PR TITLE
Upload kinematic simulator maximal coordinates

### PR DESCRIPTION
**Pull request by FabioBergonti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,84 @@
...
+
+<p align="left">
+   <a href="https://github.com/ami-iit/mystica/blob/master/LICENSE"><img src="https://img.shields.io/github/license/ami-iit/mystica" alt="Size" class="center"/></a>
+</p>
+
+**Multi bodY SimulaTor maxImal CoordinAtes**
+
+**mystica** is a library of classes and functions to simulate multibody systems.
+**mystica** computes the system evolution modeling the problem using a state definition that we define _maximal_.
+The _maximal_ representation consists of a set of _non-minimum_ variables $\textbf{q}$ complemented by holonomic constraint $g(\textbf{q})=0$.
```
Comment (traversaro) [inline README.md:11]: Most users will not have a plugin to render LaTeX, so this text will appear confusing. Probably you can stick to just use unicode characters, for example converting LaTeX to Unicode via https://www.unicodeit.net/ : 
```suggestion
The _maximal_ representation consists of a set of _non-minimum_ variables 𝐪 complemented by holonomic constraint g(𝐪) = 0.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,84 @@
...
+git clone https://github.com/ami-iit/mystica.git
+cd mystica
+```
+
+2. Run in **matlab** the function [`install()`](install.m).
+``` matlab
+install()
+```
+The function [`install()`](install.m) downloads [`mambaforge`](https://github.com/conda-forge/miniforge#mambaforge). [`mambaforge`](https://github.com/conda-forge/miniforge#mambaforge) is a package manager that downloads and configures our dependencies in conda enviroment called `mystica`.\
+If you already have [`mambaforge`](https://github.com/conda-forge/miniforge#mambaforge) configured, you can call [`install()`](install.m) function defining `mambaforge_prefix` value:
```
Comment (traversaro) [inline README.md:55]: Personal taste/nitpick: I would stick to the simplest installation option in the README, and leave the installation for people that already have mambaforge installed in another page to keep the README as simple as possible.

File: +mystica/nomenclature.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+# Nomenclature
+## Variables
```
Comment (traversaro) [inline +mystica/nomenclature.md:2]: Is this document linked or mentioned somewhere? Furthermore, I would document here the software necessary to install to render LaTeX formulas in GitHub Markdown files.

File: README.md (hunk 2 above)
Comment (FabioBergonti) [reply README.md]: @traversaro 
What do you think of using a [`disclosure widget`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details?retiredLocale=it) to hide the content?
https://github.com/ami-iit/mystica/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R55-R59

Comment (traversaro) [reply README.md]: Good idea.

**Review by traversaro — APPROVED** [review]

File: +mystica/nomenclature.md (hunk 3 above)
Comment (FabioBergonti) [reply +mystica/nomenclature.md]: > Is this document linked or mentioned somewhere?

No! I added a pointer in the readme
https://github.com/ami-iit/mystica/pull/1/commits/394bc767a13d922b7ef002adcc960d90afce7207#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R82-R83

> Furthermore, I would document here the software necessary to install to render LaTeX formulas in GitHub Markdown files.

You are right!
https://github.com/ami-iit/mystica/pull/1/commits/394bc767a13d922b7ef002adcc960d90afce7207#diff-30f1590c9d25c9533cc089fcdfed706a2b237024223e18ca16db6d857b2de656R4-R5

**Comment by FabioBergonti** [comment]
Thanks a lot @traversaro 
Merging 🎉

**Pull request merged**